src/components/header/HeaderLinks.vue

```diff
@@ -0,0 +1,65 @@
+<script setup lang="ts">
+interface Link {
+  id: number;
+  title: string;
+  url: string;
+}
+
+const links: Link[] = [
+  { id: 1, title: "Movies", url: "#" },
+  { id: 2, title: "Screenings", url: "#" },
+  { id: 3, title: "Contact Us", url: "#" },
+];
+
+// TO BE CONNECTED TO router.path WHEN WE HAVE THOSE PAGES
+const navbarItemClass = (link: Link) => {
+  return [
+    "navbar__item",
+    { "navbar__item--active": link.title === "Screenings" },
+  ];
```

Comment:
I know this is all temporary code but as a tip: we could use this in the template like so too, don't be afraid of multiline definitions 😄 :

```vue
<template>
  <ul class="navbar">
    <li
      :key="link.id" 
      v-for="link in links"
      :class="[
        'navbar__item',
        {
          'navbar__item--active': link.title === 'Screenings'
        }
      ]"
    >
      ...
    </li>
  </ul>
</template>
```